hybrid/decomposers.py

```diff
@@ -422,6 +428,91 @@ def next(self, state, **runopts):
 
         return state.updated(subproblem=subbqm)
 
+class SublatticeDecomposer(traits.ProblemDecomposer, traits.SISO, Runnable):
+    """Selects a lattice structured subproblem creating associated 
+    ``subproblem``, and ``embedding`` state fields.
+    This decomposer uses the state fields ``bqm``, ``origin_embeddings`` and 
+    ``problem_dims`` which must be compatible (in particular, through geometric
+    variable keying). 
```

Comment:
If your lattice is a cubic lattice, your embedding and your problem has to have variables specified by tuples (x,y,z). Usual geometric coordinates.
If your lattice is pegasus or chimera, they have to use pegasus nice coordinate tuples (c,x,y,u,l) or chimera tuples (x,y,u,l). This could probably be stated more clearly. However, if we generalize to other lattices it will become a zoo, so would be nice to have a concise statement

Reply:
Removed ambiguous content, relates to workflows and not this specific function